### PR TITLE
fix: IsUrl for 2nd version of pydantic

### PR DESCRIPTION
**Pull request by mishaga**
Closes issue #72

### CODE REVIEW

File: dirty_equals/_other.py [hunk 1]
```diff
@@ -260,13 +262,15 @@ def equals(self, other: Any) -> bool:
             parsed = self.parse_obj_as(self.url_type, other)
         except self.ValidationError:
             raise ValueError('Invalid URL')
+
+        equal = parsed == other if self.pydantic_version < '2' else parsed.unicode_string() == other
```
Comment (samuelcolvin) [inline dirty_equals/_other.py:266]: we'll have problems when pydantic hits V10!

**Comment by samuelcolvin** [comment]
thanks so much.

**Comment by alexmojaki** [comment]
> Closes issue #72

Did the word 'issue' throw off github?

File: dirty_equals/_other.py (hunk 1 above)
Comment (mishaga) [reply dirty_equals/_other.py:266]: I fixed this problem here: https://github.com/samuelcolvin/dirty-equals/pull/80 😄

**Pull request merged**